### ISSUE TITLE
При попытке установить EDT ничего не происходит. 

### ISSUE THREAD

**Issue by o-homyak** [1C-Company/1c-edt-issues#1492]
### Описание ошибки

Данный пост создан по рекомендации канала: https://t.me/e1c_edt
где не было найдено решение проблемы и направлено на создание ошибки на гит.

Полностью весь софт был скачан из сайта вендора.
https://releases.1c.ru/version_file?nick=DevelopmentTools10&ver=2023.3.6&path=DevelopmentTools%5c2023_3_6%5c1c_edt_distr_offline_2023.3.6_33_windows_x86_64.zip

https://releases.1c.ru/version_file?nick=Liberica11FullJDK&ver=11.0.20%2b8&path=Liberica11FullJDK%5c11_0_208%5cbellsoft_jdk11.0.20_8_windows_amd64_full.msi

Начал из онлайн установщика - ничего не происходит. Скачал установщик офлайн, ошибка осталась, а именно ничего попросту не происходит не открывается абсолютно ничего... Полсекудное раздумие системы и ничего.
Установка EDT первая.

### Как воспроизвести

Собственно в описании ошибки, не запускается установка ПО.

### Скриншоты

![2024-08-19_16-04-54](https://github.com/user-attachments/assets/3a38825a-f31d-44b4-90b2-d938f37c0f9b)
Попытки запуска из консоли под правами администратора.

![2024-08-19_16-06-47](https://github.com/user-attachments/assets/d1e2753c-553c-453f-a94b-98af76a3b0d8)
Характеристики серверной машины. НЕ используется виртуалка, физический сервер.

![2024-08-19_15-42-31](https://github.com/user-attachments/assets/72eeb7cc-b60f-4bbd-a9f8-8a8b7f90296c)
Перечень установленного ПО на сервере.

![2024-08-19_17-29-37](https://github.com/user-attachments/assets/0637d2f6-c14b-4797-94de-a054ca5e49a2)
Добавлено явно исключение в защитник.

![2024-08-19_18-09-18](https://github.com/user-attachments/assets/7f0ca214-0137-43a0-8565-b1492f0a06a8)
Отключено использование фаервола.

![2024-08-19_16-46-16](https://github.com/user-attachments/assets/ae492035-0d6b-4005-986a-d9c91e702d96)
![2024-08-19_16-42-45](https://github.com/user-attachments/assets/1032ffb2-4610-4ff5-998f-fc415a10bb6e)
![2024-08-19_16-32-37](https://github.com/user-attachments/assets/21ab9658-c73f-4908-9861-77fe37bfd278)
Это каталоги внутри которых нет никаких логов или других файлов они попросту пустые.

### Ожидаемое поведение

Ожидается окно установщика ПО.

### Лог рабочей области

Файлы логов просто отсуствуют, только windows-события...
![2024-08-19_17-32-50](https://github.com/user-attachments/assets/046bde38-0ed0-451d-b1d6-481012e4e1d2)
![2024-08-19_17-32-14](https://github.com/user-attachments/assets/9bbb1093-f4fb-44e3-b85f-62800c07191a)

### Версия 1С:EDT

1C:Enterprise Development Tools, версия 2023.3.6

### Операционная система

Windows

### Версия платформы 1С:Предприятие 8

8.3.25.1374

### Установленные плагины

Нет плагинов

**Comment by o-homyak** [comment]
ВСЕ проблема найдена. Установил SQL Management Studio и ИНСТАЛЯТОР стартовал - вывод дальше делайте сами, явно на "голую" винду оно не хочет ставится...

![2024-08-20_18-17-30](https://github.com/user-attachments/assets/a8c4f6cc-c4c2-44e3-a036-4fffdf378c75)

**Comment by tretyakevich** [comment]
MVC++ Redistributable какой-то из версий. Скорее всего оно и было нужно.

**Comment by tretyakevich** [comment]
> Это да просто хотелось бы увидить данное поведение в документации как минимум.

Да, выясняем с коллегами сейчас, так ли это, и какие конкретно версии нужны. Как минимум документацию расширим, если подтвердится

**Comment by vozhukov** [comment]
![image](https://github.com/user-attachments/assets/659e3510-df3d-454a-8702-1d46aab48661)

Windows 11 IoT Корпоративная LTSC
24H2
26100.1457

[Технологическая платформа 8.3](https://releases.1c.ru/project/Platform83)| 8.3.25.1374

[Liberica 11 Full JDK](https://releases.1c.ru/project/Liberica11FullJDK)

Аналогичная проблема, была установлена MVC++ 17версии, инсталлятор не запускался, поставил MVC++ 15 версию и запуск пошел

**Comment by onecdev** [comment]
Аналогичная проблема была при установке EDT2024.1.3 на чистую систему - после установки MVC++ запуск пошел

`winget search --name c++
winget add  Microsoft.VCRedist.2015+.x64
winget search --name Zulu 17
winget add  Azul.Zulu.17.JDK,  Azul.Zulu.17.JRE, Azul.ZuluFX.17.JRE,  Azul.ZuluFX.17.JDK`
